Kconfig.zephyr

```diff
@@ -450,6 +478,14 @@ config MCUBOOT_GENERATE_CONFIRMED_IMAGE
 	  The existence of bin and hex files depends on CONFIG_BUILD_OUTPUT_BIN
 	  and CONFIG_BUILD_OUTPUT_HEX.
 
+config MCUBOOT_TARGET_IMAGE
+	int"Selects image slot for which application will be build"
```

Comment:
add a space after the keyword, typo: `build` -> `built`

Reply:
Option will be remove; this is leftover from experiments.